calplus/tests/unit/v1/compute/drivers/test_openstack_driver.py

```diff
@@ -0,0 +1,399 @@
+""" OpenstackDriver for Compute
+    based on BaseDriver fror Compute Resource
+"""
+
+
+import mock
+from keystoneauth1.exceptions.base import ClientException
```

Comment:
pep8 ordering, all external lib import must be separated into another code block, while internal import in another code block.